src/docs/demos/pagination/pagination-demo.html

```diff
@@ -0,0 +1,57 @@
+<div class="mb-4" ng-init="currentPageBasic10 = 1">
+    <h6>Basic (Current Page: {{currentPageBasic10}})</h6>
+    <ngbs-pagination items-per-page="10" total-items="100" on-page-change="currentPageBasic10 = currentPage"></ngbs-pagination>
+</div>
+
+<div class="mb-4">
```

Comment:
sounds good to me - crazy values lead to crazy behaviour, nothing wrong with that!

but if you want feel free to leave in just a hardcoded example, you made it very easy to understand